COMETwebapp/Components/ModelDashboard/ParameterValues/ParameterDomainProgress.razor

```diff
@@ -36,7 +36,7 @@ Copyright (c) 2023 RHEA System S.A.
 				</div>
 			</div>
 			@{
-				var publishedValues = Math.Round((double)this.ValueSets.Count(d => d.Published.SequenceEqual(d.ActualValue))) * 100 / this.ValueSets.Count();
+				var publishedValues = Math.Round((double)this.ValueSets.Count(d => d.Published.SequenceEqual(d.ActualValue)) * 100 / this.ValueSets.Count());

```

Comment:
Instead of parsing to double, Can't you multiply the enumerator by 100.0? That should do the same